webapp/shop/cred/views.py

```diff
@@ -708,3 +714,42 @@ def issue_badges(trueability_api, credly_api, **kwargs):
             return (flask.jsonify(new_badge), 500)
     # 403 Forbidden. Request was valid but the server is refusing action
     return flask.jsonify({"status": "badge_not_issued"}), 403
+
+
+@shop_decorator(area="cred", permission="user", response="json")
```

Comment:
Can we make this one endpoint that returns all by default, but allows you to send a `type="keys|exams"` param if you want only one or the other?

Reply:
Done